src/NorthwindWebsite/Filters/LoggingActionFilter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using NorthwindWebsite.Core.ApplicationSettings;
+
+namespace NorthwindWebsite.Filters;
+
+public class LoggingActionFilter : IActionFilter
```

Comment:
You can just call if `LogginFilter` because your filter will be binded to actions under ther hood